pkg/webhook/blackduck.go

```diff
@@ -52,6 +53,18 @@ func (ow *OperatorWebhook) blackduckCustomResource(ar v1beta1.AdmissionReview) *
 		if strings.Compare(current.Spec.Namespace, bd.Spec.Namespace) != 0 {
 			return ow.returnError("Namespace cannot be modified")
 		}
+
+		if bd.Spec.ExternalPostgres.PostgresHost == "" {
```

Comment:
What about for internal postgres password validation?

Reply:
That's what lines 59-61 are checking.  I don't understand the comment.

Reply:
Done, good catch!